docs/vtpm.md

```diff
@@ -0,0 +1,122 @@
+# vTPM
+vtpm in Cloud-Hypervisor is emulated while using ```swtpm``` in the backend.
+
+Current implementation only supports TPM ```2.0``` version. At the moment only
+```CRB Interface``` is implementation. This interface is described in
+[TCG PC Client Platform TPM Profile Specification for TPM 2.0, Revision 01.05 v4](https://trustedcomputinggroup.org/wp-content/uploads/PC-Client-Specific-Platform-TPM-Profile-for-TPM-2p0-v1p05p_r14_pub.pdf).
+
+
+## Usage
+`--vtpm-socket`, an optional argument, can be passed to enable vtpm device.
```

Comment:
I would prefer a more generic option here. A simple `--tpm` is what I would expect, which would accept `socket=` as a parameter. And if someone wants to plug a different type of emulated TPM in the future, we might add a different parameter.